Codeinwp/neve
# Fix/css vars case

### Summary
After the discussion on the themes-HQ channel, we decided to go with renaming the CSS variables to better support PageSpeed environments,
This PR changes CSS variables' names from camel case to lowercase.

### Will affect the visual aspect of the product

NO

### Screenshots 

### Test instructions
1. Test that with the variable renaming things look the same.
2. Test Starter Sites import.
3. Check with sites imported prior to these changes.


Closes #3248.
